open-metadata-implementation/governance-servers/open-lineage-services/open-lineage-services-client/src/main/java/org/odpi/openmetadata/governanceservers/openlineage/client/util/Graphs.java

```diff
@@ -0,0 +1,20 @@
+/* SPDX-License-Identifier: Apache 2.0 */
+/* Copyright Contributors to the ODPi Egeria project. */
+package org.odpi.openmetadata.governanceservers.openlineage.client.util;
+
+public enum Graphs {
+
+    main("main"),
```

Comment:
Because they are constants, the names of an enum type's fields are in uppercase letters.
Please see:
[https://docs.oracle.com/javase/tutorial/java/javaOO/enum.html](url)